Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM node:0.12.5
+RUN apt-get update && apt-get install -y --no-install-recommends python3-minimal python3-virtualenv python3-pip python3-dev python3-pil
+RUN python3 -m virtualenv -p /usr/bin/python3 --system-site-packages /var/www/shoop/venv
+RUN /var/www/shoop/venv/bin/pip install -U pip
+ADD . /var/www/shoop/working_copy
+RUN /var/www/shoop/venv/bin/pip install /var/www/shoop/working_copy
+RUN SHOOP_WORKBENCH_DISABLE_MIGRATIONS=1 /var/www/shoop/venv/bin/python -m shoop_workbench migrate
+RUN /var/www/shoop/venv/bin/python -m shoop_workbench shoop_populate_mock --with-superuser=admin
+RUN apt-get clean && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
```

Comment:
Move the apt-get clean && rm -rf /var/lib/apt/lists/* part to same RUN command that does the "apt-get update" and "apt-get install". That'll make the intermediate image smaller.

Also move the rm -rf /tmp/* /var/tmp/* to same RUN command that leaves the cruft in /tmp and /var/tmp. Is it pip? Btw. are you sure that something really leaves cruft to both of these?